akka-cluster/src/multi-jvm/scala/akka/cluster/ClusterDeathWatchSpec.scala

```diff
@@ -173,8 +173,10 @@ abstract class ClusterDeathWatchSpec
       }
       enterBarrier("remote-watch")
 
-      // second and third are already removed
-      awaitClusterUp(first, fourth, fifth)
+      runOn(fifth) {
+        cluster.join(first)
+      }
+      enterBarrier("fifth-joined")
```

Comment:
Hmm, no need to wait for the fifth node to have joined before continuing?

Reply:
this might have to change when https://github.com/akka/akka/pull/27126 has been merged so I'm not so sure we should change this

Reply:
@raboof this test was updated and a bit of the expectation around that line have changed in https://github.com/akka/akka/pull/27126 which is indeed now merged.